test/e2e/util/job.go

```diff
@@ -211,10 +212,16 @@ func CreateJobInner(ctx *TestContext, jobSpec *JobSpec) (*batchv1alpha1.Job, err
 			restartPolicy = task.RestartPolicy
 		}
 
+		maxRetry := task.MaxRetry
+		if maxRetry == 0 {
+			maxRetry = -1
+		}
+
 		ts := batchv1alpha1.TaskSpec{
 			Name:     name,
 			Replicas: task.Rep,
 			Policies: task.Policies,
+			MaxRetry: maxRetry,
```

Comment:
Suggest to supply some introduction and examples about this field to make users and developers have a better understanding about that.

Reply:
already added e2e: "job level LifecyclePolicy, Event: TaskFailed; Action: TerminateJob"

Reply:
> already added e2e: "job level LifecyclePolicy, Event: TaskFailed; Action: TerminateJob"

THX. Perhaps add a documentation under  docs/ is better.